kubernetes/linux/main.sh

```diff
@@ -288,6 +288,27 @@ fi
 export CLOUD_ENVIRONMENT=$CLOUD_ENVIRONMENT
 echo "export CLOUD_ENVIRONMENT=$CLOUD_ENVIRONMENT" >> ~/.bashrc
 
+# Copying over CA certs for airgapped clouds. This is needed for Mariner vs Ubuntu hosts.
+# We are unable to tell if the host is Mariner or Ubuntu,
+# so both /anchors/ubuntu and /anchors/mariner are mounted in the yaml.
+# One will have the certs and the other will be empty.
+# These need to be copied to a different location for Mariner vs Ubuntu containers.
+# OS_ID here is the container distro.
+# Adding Mariner now even though the elif will never currently evaluate. 
+if [ $CLOUD_ENVIRONMENT == "usnat" ] || [ $CLOUD_ENVIRONMENT == "ussec" ]; then
+  OS_ID=$(cat /etc/os-release | grep ^ID= | cut -d '=' -f2)
```

Comment:
As per manpage - https://www.freedesktop.org/software/systemd/man/os-release.html.  Format of the ID kv pair can have quotes and also it seems optional.  Suggestions 1. probable we should have default behavior (ubuntu) in such cases or echo error message if the OS_ID either "ubuntu" or "mariner" which  will help us debug from our telemetry if we get into such situation. 
ID=
A lower-case string (no spaces or other characters outside of 0–9, a–z, ".", "_" and "-") identifying the operating system, excluding any version information and suitable for processing by scripts or usage in generated filenames. **If not set, a default of "ID=linux" may be used**. Note that even though this string may not include characters that require shell quoting, quoting may nevertheless be used.

**Examples: "ID=fedora", "ID=debian".**

Reply:
Thanks Ganga. Changed this to trim for quotes and to default to ubuntu and echo an error message if it's neither